src/App.tsx

```diff
@@ -1,73 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Autocomplete } from './Components/Autocomplete';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedUser, setSelectedUser] = useState<Person | null>(null);
+
+  const handleSelectedUser = (user: Person | null) => {
+    setSelectedUser(user);
+  };
 
   return (
     <div className="container">
       <main className="section is-flex is-flex-direction-column">
         <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
+          {selectedUser
+            ? `${selectedUser.name} (${selectedUser.born} - ${selectedUser.died})`
```

Comment:
Consider using [destruction assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to the selectedUser object